src/transports/tcp/TcpConnectionFactory.cpp

```diff
@@ -80,12 +80,4 @@ void TcpConnectionFactory::connect(OnDuplexConnectionConnect cb) {
       });
 }
 
-std::unique_ptr<DuplexConnection>
-TcpConnectionFactory::createDuplexConnectionFromSocket(
```

Comment:
I have deleted this function in my previous PR and @lehecka has informed that this function needs to stay. I am not sure if the concern of him still exists.